MainProject/Program.cs

```diff
@@ -6,29 +6,38 @@ internal class Program
     {
         static void Main(string[] args)
         {
-            Company optioCompany = new Company();
-            optioCompany.name = "Optio";
-            optioCompany.foundationYear = 1900;
+            var optioCompany = new Company("Optio", 1900, "LT001");
             optioCompany.PrintCompanyName();
 
-            Company dellCompany = new Company { name = "Dell", foundationYear = 2015 };
+            var dellCompany = new Company("Dell", 2015, "LT002");
             dellCompany.PrintCompanyName();
 
-            Shop seikShop = new Shop("Seik accossories", 2015, 5, "Vilnius");
-            Company seikCompany = new Company("Seik", 2014, seikShop);
+            var seikShop = new Shop("Seik accossories", 2015, 5, "Vilnius");
+            var seikCompany = new Company("Seik", 2014, seikShop, "LT003");
             seikCompany.PrintAllShops();
+            seikShop.GetShopInfo();
```

Comment:
Method returns string and it makes no sense in this line of code - no info we get in console